datafusion/core/src/physical_optimizer/aggregate_statistics.rs

```diff
@@ -173,6 +173,23 @@ fn take_optimizable_column_and_table_count(
     None
 }
 
+fn unwrap_min(agg_expr: &dyn AggregateExpr) -> Option<&AggregateFunctionExpr> {
```

Comment:
As part of a follow on PR, I think we should change this optimizer to use some method on AggregateFunctionExpr rather than the name

So something like 

```rust

impl AggregateFunctionExpr {

 /// Return the value of the aggregate function if there are no rows in
 /// 
 /// If the optimizer knows the input to an aggregation operation has
 /// no rows then it will replace the aggregation with a constant
 /// 
 /// If the value for 0 input rows is not known, returns None (the default)
 fn zero_row_value(&self) -> Option<ScalarValue> { None }
...
}
```

We can do this as a follow on PR